hermeznetwork/hermez-node
# Remove priceupdater

Closes #1051

### What does this PR does?

This PR removes the priceupdater functionality.
Now, the hermez node needs the priceupdater external service to work properly.

### How to test?

Run the hermez node and check that the prices are not updated until you run the priceupdater service

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
